apps/etterlatte-saksbehandling-ui/client/src/components/person/sakOgBehandling/EtteroppgjoerForbehandlingListe.tsx

```diff
@@ -34,6 +34,9 @@ function EtteroppgjoerForbehandlingTabell({
     )
   }
 
+  // TODO toggle visning av kopier eller ikke?
+  forbehandlinger = forbehandlinger.filter((forbehandling) => forbehandling.kopiertFra == null)
```

Comment:
Dette bør være en ny variabel, siden man ikke bør endre på props i React
```suggestion
  const relevanteForbehandlinger = forbehandlinger.filter((forbehandling) => forbehandling.kopiertFra == null)
```
f.eks. (og endre variabelnavnet lengre nede)